src/newcharacter.cpp

```diff
@@ -1532,11 +1539,47 @@ int set_scenario(WINDOW *w, player *u, int &points)
         mvwprintz(w_location, 0, 0, COL_HEADER, _("Scenario Location:"));
         wprintz(w_location, c_ltgray, _("\n"));
         wprintz(w_location, c_ltgray, _(sorted_scens[cur_id]->start_name().c_str()));
+
+        mvwprintz(w_flags, 0, 0, COL_HEADER, _("Scenario Flags:"));
+        wprintz(w_flags, c_ltgray, _("\n"));
+
+        if ( sorted_scens[cur_id]->has_flag("SPR_START")) {
+            wprintz(w_flags, c_ltgray, _("Spring start"));
+            wprintz(w_flags, c_ltgray, _("\n"));
+        } else if ( sorted_scens[cur_id]->has_flag("SUM_START")) {
+            wprintz(w_flags, c_ltgray, _("Summer start"));
+            wprintz(w_flags, c_ltgray, _("\n"));
+        } else if ( sorted_scens[cur_id]->has_flag("AUT_START")) {
+            wprintz(w_flags, c_ltgray, _("Autumn start"));
+            wprintz(w_flags, c_ltgray, _("\n"));
+        } else if ( sorted_scens[cur_id]->has_flag("WIN_START")) {
+            wprintz(w_flags, c_ltgray, _("Winter start"));
+            wprintz(w_flags, c_ltgray, _("\n"));
+        }
+
+        if ( sorted_scens[cur_id]->has_flag("INFECTED") ) {
+            wprintz(w_flags, c_ltgray, _("Infected player"));
+            wprintz(w_flags, c_ltgray, _("\n"));
+        }
+        if ( sorted_scens[cur_id]->has_flag("BAD_DAY") ) {
+            wprintz(w_flags, c_ltgray, _("Drunk and sick player"));
+            wprintz(w_flags, c_ltgray, _("\n"));
```

Comment:
You really don't need to mark those formatting-only strings for translation, there's nothing the translators can do with them as they don't have any context (each `_("\n")` in all the source files is translated to the same).

But it's good that you keep them out of the translated string, so I've heard.